### PR TITLE
CQ-4347866 - Support AF2 Template creation

### PR DESCRIPTION
**Pull request by pankaj-parashar**
- Adding AF2 template support creation using template console
- Policy names updated, policy nodes recreated as per resource type, existing template updated accordingly
- Adding accessControlHandling for including acls for template types

We can release this template type to public archetype repo (some modification will be required) along with core component, and ootb af2 template.

With this change, the following changes are visible.
User can create a new af2 template now on template console.

image

Now, on template console template card show label - Adaptive Form V2 (this is read from template type title)

image

After fixing of policies node structure, now, in structure layer, allowed components are visible

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/templates/af-blank-v2/.content.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <jcr:content
         cq:lastModified="{Date}2021-08-31T15:15:47.862+05:30"
         cq:lastModifiedBy="admin"
-        cq:templateType="/conf/core-components-it/settings/wcm/template-types/af-blank-v2"
+        cq:templateType="/conf/core-components-it/settings/wcm/template-types/af-page-v2"
         jcr:description="AF v2 Template for Core component examples"
         jcr:primaryType="cq:PageContent"
         jcr:title="AF Blank V2"
```
Comment (nit23uec) [inline it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/templates/af-blank-v2/.content.xml:10]: can we rename it as `Blank (Core Components)`

**Comment by StakshiGarg** [comment]
@pankaj-parashar : lets get this closed... thanks

File: it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/templates/af-blank-v2/.content.xml (hunk 1 above)
Comment (pankaj-parashar) [reply it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/templates/af-blank-v2/.content.xml]: Fixed.

Comment (pankaj-parashar) [reply it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/templates/af-blank-v2/.content.xml]: @nit23uec, Removed (Core Components), as suggested by Anurag, it was also coming from template type string as well.

**Pull request closed without merging**